hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -14,96 +14,109 @@ Please see the [boot EEPROM](booteeprom.md) page for more information about the
+|  0x2  |  NETWORK        |  Network boot - See [Network boot server tutorial](bootmodes/net_tutorial.md)                                            |
+|  0x3  |  RPIBOOT        |  RPIBOOT - See [usbboot](https://github.com/raspberrypi/usbboot)                                                         |
+|  0x4  |  USB-MSD        |  USB mass storage boot - See [USB mass storage boot](bootmodes/msd.md)                                                   |
+|  0x5  |  BCM-USB-MSD    |  USB 2.0 boot from USB Type-C socket or USB Type-A socket on CM4 IO board.     (since 2020-12-14)                        |
+|  0x6  |  NVME           |  Boot from an NVMe SSD connected to the PCIe socket (CM4 only). See [NVMe boot](bootmodes/nvme.md) for more details      |
+|  0xe  |  STOP           |  Stop and display error pattern. A power cycle is required to exit this state.                                           |
+|  0xf  |  RESTART        |  Restart from the first boot-mode in the BOOT_ORDER field i.e. loop                                                      |
+
+`RPIBOOT` is intended for use with Compute Module 4 to load a custom debug image (e.g. a Linux RAM-disk) instead of the normal boot. This should be the last boot option because it does not currently support timeouts or retries.
+
+##### `BOOT_ORDER` examples
+
+| BOOT_ORDER | Description                                                                    |
+| -----------|--------------------------------------------------------------------------------|
+| 0xf41      | Try SD first, followed by USB-MSD then repeat (default if BOOT_ORDER is empty) |
+| 0xf14      | Try USB first, followed by SD then repeat                                      |
+| 0xf21      | Try SD first, followed by NETWORK then repeat                                  |
+
+<a name="MAX_RESTARTS"></a>
 ### MAX_RESTARTS
 If the RESTART (`0xf`) boot mode is encountered more than MAX_RESTARTS times then a watchdog reset is triggered. This isn't recommended for general use but may be useful for test or remote systems where a full reset is needed to resolve issues with hardware or network interfaces.
 
 Default: `-1` (infinite)  
 
+<a name="SD_BOOT_MAX_RETRIES"></a>
 ### SD_BOOT_MAX_RETRIES
 The number of times that SD boot will be retried after failure before moving to the next boot mode defined by `BOOT_ORDER`.  
 `-1` means infinite retries.
 
 Default: `0`  
 
+<a name="NET_BOOT_MAX_RETRIES"></a>
 ### NET_BOOT_MAX_RETRIES
 The number of times that network boot will be retried after failure before moving to the next boot mode defined by `BOOT_ORDER`.  
 `-1` means infinite retries.
 
 Default: `0`  
 
+<a name="DHCP_TIMEOUT"></a>
 ### DHCP_TIMEOUT
 The timeout in milliseconds for the entire DHCP sequence before failing the current iteration.
 
 Minimum: `5000`  
 Default: `45000`  
 
+<a name="TFTP_REQ_TIMEOUT"></a>
```

Comment:
Should be `"DHCP_REQ_TIMEOUT"` to match the header

Reply:
done